docs/CHANGELOG.asciidoc

```diff
@@ -106,6 +107,31 @@ Review important information about the {kib} 8.x releases.
 
 include::upgrade-notes.asciidoc[]
 
+[[release-notes-8.19.3]]
+== {kib} 8.19.3
+
+The 8.19.3 release includes the following fixes.
+
+[float]
+[[fixes-v8.19.3]]
+=== Fixes
+Alerting::
+* Fixes an issue that prevents reports from showing in the Reporting UI when they're created in a non-default {kib} space ({kibana-pull}230876[#230876]).
+Canvas::
+* Fixes a rendering issue that affected progress elements in Canvas ({kibana-pull}232432[#232432]).
+Dashboards and Visualizations::
+* Ensures passing of the `allowLeadingWildcards` advanced setting in Lens ({kibana-pull}232609[#232609]).
```

Comment:
We need to mention that this is for ES|QL Lens charts and not for every Lens chart as they were already supporting it